src/main/kotlin/rx/lang/kotlin/observables.kt

```diff
@@ -107,3 +107,5 @@ fun <T, R> List<Observable<T>>.zip(zipFunction: (args: List<T>) -> R): Observabl
  * Returns an Observable that emits the items emitted by the source Observable, converted to the specified type.
  */
 inline fun <reified R : Any> Observable<*>.cast(): Observable<R> = cast(R::class.java)
+
+fun<T> observableDefer(x: T) = Observable.defer { Observable.just(x) }
```

Comment:
I don't want to calculate `x` right away so I use `defer` outside of `just`. Does my understanding correct?